tools/harness-thci/OpenThread.py

```diff
@@ -1783,13 +1771,12 @@ def configBorderRouter(
             if P_Prefix is None:
                 P_Prefix = 0xfd007d037d037d03
 
-            P_Prefix = '%016x' % P_Prefix
         else:
             # TestHarness 1.1 converts 2001000000000000 to "2001000000000000" (it's wrong, but not fixed yet.)
             P_Prefix = str(P_Prefix)
             int(P_Prefix, 16)
 
-        prefix = self.__convertIp6PrefixStringToIp6Address(P_Prefix)
+        prefix = self.__convertIp6PrefixStringToIp6Address(str(P_Prefix))
```

Comment:
str(P_Prefix) is different from `'%016x' % P_Prefix`

Is Harness using a different logic now?

Reply:
I haven't used that logic previously.  But I can tell you that the test wasn't passing for me until I made this change.

Reply:
The failing tests here were Border Router 7.1.1 through 7.1.5.  All failed without the above changes.

Reply:
I think @edmont added the old logic.